### ISSUE TITLE
Current nightly versions trigger `unexpected_cfgs` warning.

### ISSUE THREAD

**Issue by sirno** [ExpHP/npyz#72]
There has been a change in newer compiler versions causing `unexpected_cfgs` to be reported in derive macros  (https://github.com/rust-lang/rust/pull/132577).

The warning is triggered here:

https://github.com/ExpHP/npyz/blob/55f210eb5c1b6d8258ceba6c502422d95cb08d60/derive/src/lib.rs#L256

`cargo clippy` is about to remove its feature setting behavior as posted in https://blog.rust-lang.org/2024/02/28/Clippy-deprecating-feature-cargo-clippy.html

I suppose the line can simply be replaced with `#[allow(clippy:useless_attribute)]`.

**Comment by ExpHP** [comment]
Yes from that page it sounds like your suggestion is correct.  Gonna get this in and make a release tonight

**Comment by ExpHP** [comment]
![Image](https://github.com/user-attachments/assets/47d23163-90f8-4f05-8404-501a344ed148)

wat

**Comment by ExpHP** [comment]
![Image](https://github.com/user-attachments/assets/46d80230-0157-4cb2-8ac3-ec1c13524f61)

bbb-b-b-but
